var/spack/repos/spack_repo/builtin/packages/hpx/package.py

```diff
@@ -93,10 +93,11 @@ class Hpx(CMakePackage, CudaPackage, ROCmPackage):
     variant("examples", default=False, description="Build examples")
     variant("async_mpi", default=False, description="Enable MPI Futures.")
     variant("async_cuda", default=False, description="Enable CUDA Futures.")
+    variant("apex", default=False, description="Enable APEX support")
```

Comment:
Not a request, just to point out that it wasn't needed to switch to variant, imho. Since `instrumentation` is a multi-value variant, I don't see any reason why one option or the other would be preferred.

Reply:
I decided to use the variant to align it with Pika which is a fork of HPX.